bigquery/src/main/java/org/apache/zeppelin/bigquery/bigQueryInterpreter.java

```diff
@@ -0,0 +1,340 @@
+/**
+ * Copyright 2016 Google Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
```

Comment:
So, the google copyright should be completely removed here?
Where should I include the Google copyright then? Can you share an example?